### ISSUE TITLE
Normalize styling around platform differences for docstrings and arglists in var details

### ISSUE THREAD

**Issue by lread** [cljdoc/cljdoc#703]
## Currently
Here's an example of how cljdoc will show a var with arglists and docstrings that are different for Clojure and ClojureScript

![image](https://user-images.githubusercontent.com/967328/203151049-ca865146-cb13-4fdf-a805-1bea71d691f1.png)

## I think...
That we should use consistent styling for both for a clearer presentation.

I think the arglists styling is clearer and we could adopt the same style when docstrings differ.

So we'd have no special header when docstrings are the same for all platforms, but present

CLOJURE DOCSTRING
CLOJURESCRIPT DOCSTRING

Headers when they differ.

## Why so nitpicky?
Yeah, maybe seems a bit nit-picky, but consistent styling here will make #701 styling more straightforward for me.

**Comment by martinklepsch** [comment]
I'm a bit confused by the 

> but present
> 
> CLOJURE DOCSTRING
> CLOJURESCRIPT DOCSTRING
> 
> Headers when they differ.

bit. The rest of the issue read like you'd prefer to have the same hanging clj/cljs markers as for docstrings? 

Either way I agree that making this consistent is a good idea 👌

**Comment by lread** [comment]
Thanks for asking about this @martinklepsch!

Yeah, instead of the little "clj" and "cljs" prefix for docstrings, I was thinking of matching the style of the arglists header.

So for the above we'd instead have:

```
fn2
CLOJURE ARGLISTS
(fn2 a)
(fn2 a b)
CLOJURESCRIPT ARGLISTS
(fn2 a)
CLOJURE DOCSTRING
some clj specific docstring
CLOJURESCRIPT DOCSTRING
some cljs specific docstring
```

I'm toying around with it a bit. The headers are a bit wordy in some cases and steal away too much attention from the content.
This might be less header-noisy:

```
fn2
CLOJURE
 (fn2 a)
 (fn2 a b)
 some clj specific docstring
CLOJURESCRIPT
 (fn2 a)
 some cljs specific docstring
```
If only the docstring differ:

```
fn2
(fn2 a)
CLOJURE
 some clj specific docstring
CLOJURESCRIPT
 some cljs specific docstring
```

If only the arglists differ:
```
fn2
CLOJURE
 (fn2 a)
 (fn2 a b)
CLOJURESCRIPT
 (fn2 a)
Some common docstring
```

But honestly, this case of differing docstrings and arglists across platforms is pretty rare.

**Comment by martinklepsch** [comment]
Ah I think I didn't read this properly earlier 😅  I think what you outlined sounds good to me!

**Comment by lread** [comment]
Ok. Simple things take a while, yeah? :slightly_smiling_face: 

I've played around more and think the short-forms (clj and cljs as opposed to the longer header text) are a better choice.
They are consistent with other parts of the UI.
They also tell the story without so much text.

I've added some more platform-related code to cljdoc-exerciser. 
The "Before" is what things currently look like and the "After" is my work in progress.

![cljdoc-platforms](https://user-images.githubusercontent.com/967328/203443681-fdd3d6d0-86d4-439e-b0c0-12f0bc74ac46.png)

Our previous clj and cljs prefixes could be confusing because it was not always easy to tell what they applied to. With a bit of indentation, I think it is now clear.

**Comment by corasaurus-hex** [comment]
ooooh that's a huge improvement, Lee!

**Comment by lread** [comment]
Thanks so much for the feedback @corasaurus-hex, I shall ship it sometime soon.

**Comment by lread** [comment]
Sweet! Glad you like it too, @martinklepsch!
